Sources/ComposableArchitecture/ViewStore.swift

```diff
@@ -319,12 +341,6 @@ extension ViewStore where State: Equatable {
   }
 }
 
-extension ViewStore where State == Void {
-  public convenience init(_ store: Store<Void, Action>, instrumentation: Instrumentation = .shared, file: StaticString = #file, line: UInt = #line) {
-    self.init(store, removeDuplicates: ==, instrumentation: instrumentation, file: file, line: line)
```

Comment:
By always using this `==` we were essentially doing the same but still paying for the cost of scoping down and then checking the equality. Now we will just not even have to scope or compare.